packages/core/src/components/Dropdown/__tests__/__snapshots__/Dropdown.snapshot.test.js.snap

```diff
@@ -7,18 +7,18 @@ exports[`Dropdown renders correctly snapshot driver should open menu on click if
     id="dropdown-menu-id"
   >
     <span
+      aria-atomic="false"
       aria-live="polite"
-      class="css-1laao21-a11yText"
+      aria-relevant="additions text"
+      class="css-1f43avz-a11yText-A11yText"
     >
```

Comment:
all this new aria-live behavior is intended by react-select 4.0.0, does not affect us, only adds a11y